graphqlhub-schemas/src/github.js

```diff
@@ -187,6 +188,21 @@ let IssueType = new GraphQLObjectType({
 });
 
 
+let BranchType = new GraphQLObjectType({
+  name : 'GitBranch',
```

Comment:
nit - can we make this `GithubBranch` to keep in line with the rest of the names?